app/models/project.rb

```diff
@@ -55,8 +55,12 @@ def docker_repo
   # branch - The String name of the branch in question.
   #
   # Returns true if new releases should be created, false otherwise.
-  def create_releases_for_branch?(branch)
-    release_branch == branch
+  def create_releases_for_branch?(branch, service_type, service_name)
+    release_branch == branch && release_source_included?(service_type, service_name)
+  end
+
+  def release_source_included?(service_type, service_name)
```

Comment:
This was the only place I could find it, and it's just a query.

https://github.com/zendesk/samson/blob/master/app/models/webhook.rb#L18